examples/count.py

```diff
@@ -16,5 +16,9 @@
 scrollphat.set_brightness(7)
 
```

Comment:
@alexellis it doesn’t really make sense to move it up. there is no display output at that moment yet..